src/App.tsx

```diff
@@ -1,26 +1,275 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
```

Comment:
Fix it 
<img width="406" alt="image" src="https://github.com/user-attachments/assets/69a963f6-e9eb-454a-b8c5-c738c9b78f1c" />